doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -254,6 +254,12 @@ nRF Desktop
     This change results from the Bluetooth subsystem transition to the PSA cryptographic API.
     The GATT database hash calculation now requires larger stack size.
 
+  * Support for Bluetooth LE legacy pairing is no longer enabled by default, because it's not secure.
+    Using Bluetooth LE legacy pairing introduces risk of eavesdropping.
+    Supporting Bluetooth LE legacy pairing makes device vulnerable for downgrade attack.
+    The :kconfig:option:`CONFIG_BT_SMP_SC_PAIR_ONLY` is enabled by default in Zephyr.
+    If you still need to support the Bluetooth LE legacy pairing you need to disable the option in configuration.
```

Comment:
```suggestion
  * Support for Bluetooth LE legacy pairing is no longer enabled by default, because it is not secure.
    Using Bluetooth LE legacy pairing introduces a risk of eavesdropping.
    Supporting Bluetooth LE legacy pairing makes devices vulnerable for downgrade attack.
    The :kconfig:option:`CONFIG_BT_SMP_SC_PAIR_ONLY` Kconfig option is enabled by default in Zephyr.
    If you still need to support the Bluetooth LE legacy pairing, you need to disable the option in the configuration.
```

Reply:
Deleted my comments as its already covered in Pekka's suggestion.